bitnami/ejbca/README.md

```diff
@@ -0,0 +1,279 @@
+| `mariadb.master.persistence.enabled`      | Enable database persistence using PVC                                                 | `true`                                                       |
+| `mariadb.master.persistence.accessModes`  | Database Persistent Volume Access Modes                                               | `[ReadWriteOnce]`                                            |
+| `mariadb.master.persistence.size`         | Database Persistent Volume Size                                                       | `8Gi`                                                        |
+| `externalDatabase.host`                   | Host of the external database                                                         | `localhost`                                                  |
+| `externalDatabase.user`                   | Existing username in the external db                                                  | `bn_ejbca`                                                   |
+| `externalDatabase.password`               | Password for the above username                                                       | `nil`                                                        |
+| `externalDatabase.existingSecret`         | Name of an existing secret resource containing the DB password in a 'mariadb-password' key | `nil`                                                   |
+| `externalDatabase.database`               | Name of the existing database                                                         | `bitnami_ejbca`                                              |
+| `externalDatabase.port`                   | Database port number                                                                  | `3306`                                                       |
+
+The above parameters map to the env variables defined in [bitnami/ejbca](http://github.com/bitnami/bitnami-docker-ejbca). For more information please refer to the [bitnami/ejbca](http://github.com/bitnami/bitnami-docker-ejbca) image documentation.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install my-release \
+  --set ejbcaAdminUsername=admin,ejbcaAdminPassword=password,mariadb.db.password=secretpassword \
+    bitnami/discourse
+```
+
+The above command sets the EJBCA administrator account username and password to `admin` and `password` respectively. Additionally, it sets the MariaDB `bn_ejbca` user password to `secretpassword`.
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install my-release -f values.yaml bitnami/ejbca
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Configuration and installation details
```

Comment:
Inside this section, we try to avoid commands like `helm install ...` or `helm upgrade ...` because there are some places where users are installing the chart without a CLI, like marketplaces.
In those cases, this section of the README is rendered in the marketplace, so if possible try to explain how to set up the replication but avoiding specific commands, you can check other READMES as an example if you want

Reply:
I have re-written it in a more neutral and general way 😁